### DISCUSSION TITLE
Azure for Students

### DISCUSSION THREAD

**Discussion by krishna9919** [community/community#22391] [New to GitHub]
<p>I had applied for GitHub student and my account got confirmed. I requested for the Microsoft Azure offer code and got it. But I am unable to redeem the offer code on Microsoft azure for student. Please help me with this as soon as possible.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/krishna9919">@krishna9919</a>!</p>
<p>If you haven’t done so already, I would recommend <a href="https://support.microsoft.com/en-us/help/4026305/sql-contact-microsoft-azure-support" rel="nofollow noopener">contacting Microsoft Azure support</a> first to troubleshoot why the code you received isn’t working. They should have more information on their side about why the code is failing.</p>
<p>If that doesn’t work, I would recommend <a href="https://github.com/contact" rel="nofollow noopener">contacting GitHub Support directly</a> so that we can pass your inquiry to the department that manages the Student Developer Pack.</p>
<p>Thanks! </p>

**Comment by Emmytheo** [comment]
<p>Please tell me how you got to where you put in the code for them.<br>
I’m having the same issue.<br>
It’s opens my azure account but I can’t find where to input the code to start with.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/emmytheo">@emmytheo</a>, did you contact Azure support? Were they able to help at all? </p>

**Reply by volvetindia** [reply]
No waiting for help for 2 weeks

**Comment by Lanson2232** [comment]
<p>i have Github student pack and i want use Microsoft Azure BUT i dont have a School mail or have get Any Code from github/Azure for verifying , can anyone help me?</p>

**Reply by Ayush-Kumar-web** [reply]
Same here for me

**Reply by Shofikul-Isl4m** [reply]
same here (2)

**Comment by BiswaViraj** [comment]
<p>Same here.<br>
Don’t have a school email address.<br>
Also there is no verification code available on the Github site</p>

**Comment by oplik0** [comment]
<p>Same here. The offer just redirects to <a href="https://azure.microsoft.com/en-gb/free/students/" rel="nofollow noopener">https://azure.microsoft.com/en-gb/free/students/</a> now</p>
<p>While I do have a school email, apparently it isn’t accepted because it doesn’t have .edu TLD (it has .edu.pl). I can select verification with code on Azure, but I don’t have the code from GitHub, so I’m unable to verify myself.</p>

**Comment by abusayed0206** [comment]
<p>Same… I have edu.bd domain! I don’t have/didn’t get any Verification code! If you solve your problem pls reply here &lt;3</p>

**Reply by Shofikul-Isl4m** [reply]
bro apni ki thik korte parilen? amar varsity mail nai but github student developer pack ase

**Comment by yashash-reddy** [comment]
<p>Do any one Find any solution? I have the same problem, no Verification Code available on Github benifits page and I have no .EDU mailID.</p>

**Comment by gauravmehta13** [comment]
<p>Found A Solution.<br>
Just logout from your current Microsoft Account and Login with Github from the sign in page.<br>
After that Click on the link provided by Github pack.<br>
it will verify you automatically without ANY code or .edu mail.</p>

**Comment by aishwaryasonar** [comment]
<p>It does’t work either</p>

**Comment by AlecBlance** [comment]
<p>The solution mentioned does work.<br>
Here are the steps:</p>
<ol>
<li>Open this <a href="https://aka.ms/azure4students" rel="nofollow noopener">https://aka.ms/azure4students</a>
</li>
<li>Click “Activate Now”</li>
<li>In the upper right click “Logout”</li>
<li>After logging out click “Sign in with different account”</li>
<li>Then “Login with Github” from the sign in page</li>
<li>Follow the process</li>
<li>After successful linking, it’ll ask you to agree with the terms</li>
</ol>
<p>Neither edu email nor verification code will show up.</p>
<p>Done!</p>

**Reply by zeroninefourfour** [reply]
i selected login with github,same
We could not confirm your status as a student. Click [here](https://go.microsoft.com/fwLink/p/?LinkID=2136428&clcid=0x409) to begin the academic verification process. This process may take up to a few days. Once verification is complete, please come back and sign in with GitHub to redeem Azure for Students.

**Reply by Shofikul-Isl4m** [reply]
i did it . but We could not confirm your status as a student. Click [here](https://go.microsoft.com/fwLink/p/?LinkID=2136428&clcid=0x409) to begin the academic verification process. this problem showing . i do it today .is it will solve in a few days?

**Comment by JosephHeadley1** [comment]
<p>Someone help me.</p>
<p>I tried applying for Azure, it says this -</p>
<p>We could not confirm your status as a student. Click <a href="https://go.microsoft.com/fwLink/p/?LinkID=2136428&amp;clcid=0x409" rel="noopener nofollow ugc">here</a> to begin the academic verification process. This process may take up to a few days. Once verification is complete, please come back and sign in with GitHub to redeem Azure for Students.</p>
<p>Any fix ? I am logged into Azure using the Github option, still it says the same.</p>

**Comment by Ankit-vj** [comment]
<p>did you find any solution to this dude?? , as i am also facing the same issue</p>

**Comment by Aggis15** [comment]
<p>Tried the same thing as Alec suggested. Don’t even get the GitHub option when trying to sign up for the student plan. Contacted GitHub, so I’ll see how it goes. Already contacted MSFT but got nowhere.</p>

**Comment by gripeinst** [comment]
<p>Still having the same problem here, any new solutions?</p>

**Comment by blablamukul** [comment]
<p>I had the same issue. To fix it:</p>
<ol>
<li>Unlink github account from your microsoft account.</li>
<li>Go back to the student Azure page, click on “start free”.</li>
<li>Then sign in with Github, the select an email address which is not linked to any Microsoft account.</li>
<li>Then it will ask for details for creating the new MIcrosoft account and after completion, you will be redirected to fill in details for the Azure redemption.</li>
</ol>

**Comment by shubhamsm-2000** [comment]
<p>how can I get offer code? Cuz I also got approval but not able to log in with free github developer pack?</p>

**Comment by azzeddine1977** [comment]
Any help: I ​​tried to log in to Azure using my github account, and the problem is that this message appears: We could not confirm your status as a student. Click here to begin the academic verification process. This process may take up to a few days. Once verification is complete, please come back and sign in with GitHub to redeem Azure for Students.

**Comment by jaidernunes** [comment]
Same problem here still :\ None of the fixes above worked for me
